projectile.el

```diff
@@ -2640,6 +2640,41 @@ is chosen."
 
 (projectile-commander-bindings)
 
+(defun projectile-skel-read-variable ()
+  "Prompt for a variable and return its name."
+  (completing-read "Variable: "
+                   obarray
+                   '(lambda (v)
+                      (and (boundp v) (not (keywordp v))))
+                   t))
+
+(define-skeleton projectile-skel-variable-cons
+  "Insert a variable-name and a value in a cons-cell."
+  "Value: "
+  "("
+  (projectile-skel-read-variable)
+  " . "
+  str
+  ")")
+
+(define-skeleton projectile-skel-dir-locals
+  "Insert a .dir-locals.el template."
+  nil
+  "((nil . (("
+  ("" '(projectile-skel-variable-cons) \n)
+  resume:
+  "))))")
+
+(defun projectile-edit-dir-locals ()
+  "Edit or create a .dir-locals.el file of the project."
+  (interactive)
+  (let ((file (concat (projectile-project-root) ".dir-locals.el")))
+    (find-file file)
```

Comment:
This code is a bit obscure - I'd create the file explicitly if I see it hasn't been created.